next-app/src/routes/projects/[project_code]/activities/+server.ts

```diff
@@ -66,12 +71,13 @@ export async function fetch_activities({ cookie, start_date, end_date }: Activit
 	return activity.map(transform)
 }
 
-function transform({ id, action, date, content }: LegacyActivity): Activity {
+function transform({ id, action, date, content, userRef }: LegacyActivity): Activity {
 	return {
 		id,
 		action,
 		date,
-		user: content.user,
+		user: userRef.username,
```

Comment:
was this changed in case the user's username has changed since he/she performed the activity?

Reply:
That is correct. We are replacing the former username of a deleted account with its non-personally-identifiable user ID. We want this new "username" (ID) to show up in the logs of activity left in the projects the user used to be a part of.